pyqtorch/utils.py

```diff
@@ -741,3 +743,15 @@ class SolverType(StrEnum):
 
     KRYLOV_SE = "krylov_se"
     """Uses Krylov Schrodinger equation solver"""
+
+
+def is_parametric(operation: pyq.Sequence) -> bool:
+    params = []
+    for m in operation.modules():
+        if isinstance(m, pyq.Scale):
+            params.append(m.param_name)
+
+    res = False
+    if any([isinstance(p, str) for p in params]):
+        res = True
+    return res
```

Comment:
Nice :) I think it could be useful to extend this function to work for any operation in PyQ.

Reply:
I think it should work already now for any operation - the only place where a non-fixed parameter may sit is the `Scale` block. So that's what is checked here.

Reply:
Hmm I think this would fail for something like `Sequence([RX, RY])` or even just a single RX. Ok to leave it like that for now, but in which case I would add some info in the docstring.

Reply:
Yeah, that won't work now, but the solution is easy - just check in `isinstance` for `Parametric` class as well. That should cover all the parametrized operations.